### ISSUE TITLE
[QUESTION] "No messages yet".

### ISSUE THREAD

**Issue by rockneverdies55** [RocketChat/Rocket.Chat.Android#1129]
## Description

All the channels, private groups and direct messages show "No messages yet" subtitle/description in the Chats screen. All of which contain many many messages inside.

## Devices and Versions

Your Rocket.Chat.Android version: 2.0.1
Your Rocket.Chat Server version: 0.63.1

Mobile device model and OS version: Moto G - Plus / android 7.1.1

**Comment by rafaelks** [comment]
This one duplicates this question: https://github.com/RocketChat/Rocket.Chat.Android/issues/1100#issuecomment-382204324.

Please take a look at my comment.

**Comment by rockneverdies55** [comment]
Sorry for the duplicate. I searched for "no messages yet" but github didn't bring any results so I opened a new issue.

Speaking of that I turned that feature on in the admin panel, I killed the app on the phone and restarted it but still seeing the same no messages yet messages everywhere. Maybe it will update channels list later when it re-fetches the metadata? Not a big deal just wanted to let you know.
